server/container/core/src/main/java/org/apache/james/server/core/MailImpl.java

```diff
@@ -243,6 +243,7 @@ private MailImpl(Mail mail, String newName) throws MessagingException {
         setRemoteHost(mail.getRemoteHost());
         setRemoteAddr(mail.getRemoteAddr());
         setLastUpdated(mail.getLastUpdated());
+        setErrorMessage(mail.getErrorMessage());
```

Comment:
does it fix something ?

Reply:
Yes. see later on. (File and Memory implementations)


I choose to move these fix in separate commits. For readability. Should I incorporate them in the related test commit?

Reply:
it's better when located in the changeset that requires it